### DISCUSSION TITLE
Error: Not Found https://binaries.prisma.sh/all_commits/[hash]/linux-arm64-openssl-1.0.x/libquery_engine.so.node.gz

### DISCUSSION THREAD

**Discussion by Ezehuche** [prisma/prisma#11447] [Q&A]
Hi all,

Pls I got this error running `prisma generate` on Amazon linux 2

`Error: Not Found https://binaries.prisma.sh/all_commits/2452cc6313d52b8b9a96999ac0e974d0aedf88db/linux-arm64-openssl-1.0.x/libquery_engine.so.node.gz`

Can anyone help me out with this.

Thanks

**Comment by austincrim** [comment]
Hey @Ezehuche 👋 ,

Do you have anything specified in the [`binaryTargets`](https://www.prisma.io/docs/concepts/components/prisma-schema/generators#binary-targets) field of your `generator` block in the Prisma schema?

It should look like this:
```prisma
generator client {
  provider      = "prisma-client-js"
  binaryTargets = ["linux-arm64-openssl-1.0.x", "native"]
}
```

`binaryTargets` tells Prisma which query engines to download in order to be compatible with various operating systems.

Let me know!

**Reply by vladi-stevanovic** [reply]
👋 Hello @Ezehuche I'm going through pending GitHub Discussions to check if you need any help. Is this issue still open?
Please let us know if we can investigate it further or if you've already solved it, we'd love to know how!

It would be super useful for other users facing this problem! :raised_hands:

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
